bin/stashcp

```diff
@@ -141,6 +143,19 @@ def doStashCpSingle(sourceFile, destination, cache, debug=False):
         es_send(payload)
 
 
+def parse_job_ad():
+    temp_list = {}
+    try:
+        with open(os.environ['_CONDOR_JOB_AD']) as job_file:
+            for line in job_file.readLines():
+                match = re.match('Owner|ProjectName', line, re.IGNORECASE)
+                if match:
+                    temp_list[match.group()] = line
```

Comment:
I tested it, but your regex is less prone to error. Sorry, not the best with regex haha. Where would the unit test be placed?